backend/b2/b2.go

```diff
@@ -160,7 +160,19 @@ free egress for data downloaded through the Cloudflare network.
 Rclone works with private buckets by sending an "Authorization" header.
 If the custom endpoint rewrites the requests for authentication,
 e.g., in Cloudflare Workers, this header needs to be handled properly.
-Leave blank if you want to use the endpoint provided by Backblaze.`,
+Leave blank if you want to use the endpoint provided by Backblaze.
+
+The URL provided here SHOULD have the protocol and SHOULD NOT have
+a trailing slash or specify the /file/bucket subpath rclone will
+request files with "{download_url}/file/{bucket_name}/{path}".
+
+Example:
+Use https://mysubdomain.mydomain.tld instead of:
```

Comment:
I actually found this example quite confusing as it has a lot of wrong ways of doing it! I see what you were going for, but I'd rather just have the single correct example, or maybe

Example:

    https://mysubdomain.mydomain.tld

(No trailing `/`, `file` or `bucket`)

Reply:
Resolved